webserver/views/api/v1/test/test_dataset_eval.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import absolute_import
+from webserver.testing import ServerTestCase
+from db.testing import TEST_DATA_PATH
+import db.exceptions
+import db.dataset
+import db.dataset_eval
+import db.api_key
+from utils import dataset_validator
+
+import json
+import mock
+import os
+
+class APIDatasetEvaluationViewsTestCase(ServerTestCase):
+
+    def setUp(self):
+        super(APIDatasetEvaluationViewsTestCase, self).setUp()
+
+        self.test_user_mb_name = "tester"
+        self.test_user_id = db.user.create(self.test_user_mb_name)
+        self.test_user = db.user.get(self.test_user_id)
+
+
+    @mock.patch("db.dataset_eval.get_pending_jobs_for_user")
+    def test_get_pending_jobs_for_user(self, get_pending_jobs_for_user):
+        self.temporary_login(self.test_user_id)
+
+        get_pending_jobs_for_user.return_value = [{'id': "7804abe5-58be-4c9c-a787-22b91d031489"}]
+        submit = json.dumps({})
+        resp = self.client.get("/api/v1/dataset-eval/pending-jobs-for-user", data=submit, content_type='application/json')
```

Comment:
I think you only need to set `data` if this is a `POST`. for the `GET` you should just need the url.